.github/image_licenses.yaml

```
@@ -26,9 +26,9 @@ licenses:
   docker.io/bitnami/postgresql:
     license: PostgreSQL
     licenseLink: https://www.postgresql.org/about/licence/
-  docker.io/bitnami/redis:
-    license: SSPL-1.0
-    licenseLink: https://redis.io/legal/licenses/
+  docker.io/bitnami/valkey:
+    license: Apache-2.0
```

This is actually BSD-3, see [docker hub](https://hub.docker.com/r/bitnami/valkey#:~:text=Valkey%20is%20an%20open%20source%20(BSD)%20high%2Dperformance%20key/value%20datastore%20that%20supports%20a%20variety%20workloads%20such%20as%20caching%2C%20message%20queues%2C%20and%20can%20act%20as%20a%20primary%20database.), [valkey's github](https://github.com/valkey-io/valkey?tab=License-1-ov-file) and [Wikipedia](https://en.wikipedia.org/wiki/Valkey)

```suggestion
    license: BSD-3
```